pkg/controller/installation/defaults_test.go

```diff
@@ -54,7 +54,9 @@ var _ = Describe("Defaulting logic tests", func() {
 
 	It("should not override custom configuration", func() {
 		var mtu int32 = 1500
-		var ff bool = true
+		var false_ = false
```

Comment:
In the original PR there were some additional env checks (https://github.com/tigera/operator/pull/374/files#diff-0c187240f4dee17e5defcdc433f23a29R37)